src/Winook/HookBase.cs

```
@@ -107,6 +107,17 @@ private void ReleaseAndDisposeMutex()
             }
         }
 
+        private void WaitForInputIdle()
+        {
+            try
```

Don't use try/catch here. Let the exception bubble up.